randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -209,7 +224,13 @@ Extra - asset_id: 4
   > Door to Bowling Alley
       Trivial
   > Pickup (Topmost Missile Expansion)
-      Hi-Jump Boots or Space Jump
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Knowledge (Intermediate)
+          Speed Booster and Shinespark (Beginner) and Normal Damage ≥ 36
+          # Wall jump off of Trippers
+          Ice Beam and Wall Jumps (Intermediate)
+          # IBJ
+          Morph Ball Bombs and Morph Ball and Vertical Infinite Bomb Jump (Intermediate)
```

Comment:
The bombs section should be replaced with a template

Reply:
Done.